pkg/storage/metrics.go

```diff
@@ -155,6 +155,9 @@ var (
 	metaSysCount = metric.Metadata{
 		Name: "syscount",
 		Help: "Count of system KV pairs"}
+	metaWritesPerSecond = metric.Metadata{
+		Name: "writes_per_second",
+		Help: "Approximate number of keys written per second to the store"}
```

Comment:
Should clarify whether these are "active" writes (proposals) or any Raft-applied writes.